rollup-common.js

```diff
@@ -63,7 +64,11 @@ const crossPackagePropertyMangles = {
   _element: "E",
   _options: "O",
   _template: "t",
-  _renderOptions: "R"
+  _renderOptions: "R",
```

Comment:
Can we add a comment after each field here to name the class/object they are associate with, just for bookkeeping?

Reply:
Done.

Reply:
Nice, ty!